tests/test_scenes.py

```diff
@@ -26,9 +30,15 @@ def containers():
     yield BaseScene(Mock())
 
 
-@fixture()
-def player():
-    return TestPlayer()
+def players():
+    yield TestPlayer()
+    yield TestSubclassPlayer()
+
+
+def players_and_containers():
```

Comment:
You can do this with `pytest.mark.parametrize`. [Notes at the end of the first section of the docs here](https://docs.pytest.org/en/latest/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions).

> To get all combinations of multiple parametrized arguments you can stack parametrize decorators:
> ```python
> import pytest
> 
> 
> @pytest.mark.parametrize("x", [0, 1])
> @pytest.mark.parametrize("y", [2, 3])
> def test_foo(x, y):
>     pass
> ```
> 
> This will run the test with the arguments set to x=0/y=2, x=1/y=2, x=0/y=3, and x=1/y=3 exhausting parameters in the order of the decorators.

Reply:
This is explicitely not `parametrize` because it needs to be a *different* container for each test (as opposed to reusing the container across tests)

Reply:
Going to think on that, then. Ignore for now.